### ISSUE TITLE
[Question] Any consideration of supporting Java Platform Extension

### ISSUE THREAD

**Issue by gayanper** [xorye/vscode-quarkus#709]
Given that oracle is now releasing a vscode extension with thin Netbeans LS module, is there a plan to change this extension so that it can either run this JDT.LS or NB.LS by providing the shim layer to convert the expectations of JDT.LS from NB.LS ?

**Comment by angelozerr** [comment]
Switching to NB LS means re developping all our services which collect informations like scanning all java classes from rhe classpath project to collect and compute properties used for complétion validation etc in the application.properties which is a very long task.

We did that for IntelliJ and it is a really long task. 

As our team is working hard on JDT LS we don t see any value for us to do that.

However as microprofile ls is not linked to jdt it should be possible to do that since we use microprofile ls in IntelliJ where we have implemented services with IntelliJ PsiFile instead of JDT ICompilationUnit
